cin_validator/__main__.py

```diff
@@ -79,5 +79,32 @@ def test_cmd(rule, ruleset):
     pytest.main(test_files)
 
 
+@cli.command(name="xmltocsv")
+@click.argument("filename")
+def cli_converter(filename: str):
```

Comment:
@click.argument("filename", type=click.Path(), required=True)

This works also.